pages/app-layout/embedded-view-mode.page.tsx

```diff
@@ -0,0 +1,130 @@
+    },
+  ];
+
+  const [navigationOpen, setNavigationOpen] = useState(false);
+  const [toolsOpen, setToolsOpen] = useState(false);
+  const [preferences, setPreferences] = useState<CollectionPreferencesProps.Preferences>({
+    stickyColumns: { first: 1, last: 1 },
+    visibleContent: visibleContentOptions[0].options.map(o => o.id),
+  });
+  const items = generateItems(20);
+
+  return (
+    <AppLayout
+      {...{ __embeddedViewMode: true }}
+      ariaLabels={labels}
+      breadcrumbs={
+        <BreadcrumbGroup
+          items={[
+            { text: 'Home', href: '#' },
+            { text: 'Service', href: '#' },
+          ]}
+        />
+      }
+      contentType="table"
+      navigationOpen={navigationOpen}
+      toolsOpen={toolsOpen}
+      onNavigationChange={({ detail }) => setNavigationOpen(detail.open)}
+      onToolsChange={({ detail }) => setToolsOpen(detail.open)}
+      splitPanelOpen={true}
+      splitPanel={
+        <SplitPanel header="Split panel header" i18nStrings={splitPaneli18nStrings}>
+          I need to be on top! Even on mobile!
+        </SplitPanel>
+      }
+      content={
+        <Table
+          resizableColumns={true}
+          variant="full-page"
+          stickyHeader={true}
+          footer={
+            <Box textAlign="center">
+              <Button onClick={() => setToolsOpen(!toolsOpen)}>Click me to open the tools panel</Button>
+            </Box>
+          }
+          stickyColumns={preferences.stickyColumns}
+          visibleColumns={preferences.visibleContent}
+          preferences={
+            <CollectionPreferences
+              title="Preferences"
+              confirmLabel="Confirm"
+              cancelLabel="Cancel"
+              onConfirm={({ detail }) => setPreferences(detail)}
+              preferences={preferences}
+              visibleContentPreference={{
+                title: 'Select visible columns',
+                options: visibleContentOptions,
+              }}
+              stickyColumnsPreference={{
+                firstColumns: {
+                  title: 'First column(s)',
+                  description: 'Keep the first column(s) visible while horizontally scrolling table content.',
+                  options: [
+                    { label: 'None', value: 0 },
+                    { label: 'First column', value: 1 },
+                    { label: 'First two columns', value: 2 },
+                  ],
+                },
+                lastColumns: {
+                  title: 'Stick last visible column',
+                  description: 'Keep the last column visible when tables are wider than the viewport.',
+                  options: [
+                    { label: 'None', value: 0 },
+                    { label: 'Last column', value: 1 },
+                  ],
+                },
+              }}
+            />
+          }
+          header={
+            <Header variant="awsui-h1-sticky" actions={<div style={{ height: '10vh' }} />}>
```

Comment:
Split panel toggle does not work, this header has unexpectedly large height.

Not sure if you did all of it intentionally or not, but this page is confusing

Reply:
I've simplified the page.